File: Java/account.java
public class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    /*metodo constructor de Account, "this" hace referencia a las
     * variables globales*/
    public Account(String name, String document, String email, String password){
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }
}

File: Python/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str

    """
    Metodo Constructor:
    La palabra self hace referencia a las
    variables globales. Ingresa como parametro
    """
    def __init__(self, name, document, email, password):
        self.name = name
        self.document = document
        self.email = email
        self.password = password

